### ISSUE TITLE
Color picker popup are not clearly visible in High Contrast 'Aquatic' and 'Desert' modes.

### ISSUE THREAD

**Issue by cgk1994** [tinymce/tinymce#8495]
The colors present in background color picker for formatting table cells is not clearly visible in High Contrast 'Aquatic' and 'Desert' modes.

1. Open tinyMCE editor - https://codepen.io/cgk1994/pen/JjaEZWQ
2. Insert a table.
3. Turn on High Contrast(Aquatic/Desert) using Settings->Accessibility->Contrast Themes->Aquatic/Desert.
4. Go to Table -> Table Properties -> Advanced -> Open color picker available for Border color and background color

The colors present in color pallet should be visible clearly irrespective of themes we have selected but the colors are not at all visible.
(Lot of people in our customer base are relying on these contrast themes to carry out their expected tasks on daily basis and its becoming difficult for them to format the table cells because of this issue)

![tableprops_tinyMCE1](https://user-images.githubusercontent.com/126610719/221953609-39ed0d46-9e4a-4eca-ad07-cd416266f6a3.png)

tinyMCE version - 6.0
Browser - Google chrome

**Comment by cgk1994** [comment]
@HAFRMO -- Hey, I am not sure whether you will be able to provide any insights about this issue, I am just trying my luck here as its important for our project. Is there any way to fix the above mentioned issue or any workarounds that you can suggest for us?

**Comment by cgk1994** [comment]
@HAFRMO  -- Will this issue get resolved if in case we subscribe to the premium version of tinyMCE?
